src/os/bluestore/BlueStore.h

```diff
@@ -241,12 +241,15 @@ enum {
 using bptr_c_it_t = buffer::ptr::const_iterator;
 
 extern const std::vector<uint64_t> bdev_label_positions;
+class BlueStore_debug;
 
 class BlueStore : public ObjectStore,
 		  public md_config_obs_t {
   // -----------------------------------------------------
   // types
 public:
+  friend class BlueStore_debug;
```

Comment:
Might be a good idea to mention that bs_debug is _derived_ from bs

Reply:
This is explained in BlueStore_debug.h :
```
/*
 * BlueStore_debug gathers functions that are needed for in-depth testing.
 * The functions defined here are not to be used to provide any production functionality;
 * instead, functions here are allowed to work properly only in limited context
 * provided their side effect do not corrupt wider BlueStore state.
 */
```
You think some short comment is needed here ?